pkg/validator/validation_error.go

```diff
@@ -0,0 +1,56 @@
+package validator
+
+// ValidationError is a custom error adding a Field member.
+type ValidationError struct {
```

Comment:
The `Validation` prefix seems redundant. When used it becomes `validator.ValidationError` when `validator.Error` would be enough

Reply:
Also the filename could be just `error.go`

Reply:
fixed